molecule/resources/verify.yml

```diff
@@ -42,6 +40,16 @@
             openwisp_controller.geo.tests.test_api
       changed_when: false
 
+    - name: Run integration tests when monitoring disabled
+      when: not openwisp2_monitoring
+      command: >
+        /opt/openwisp2/env/bin/python /opt/openwisp2/manage.py test --keepdb \
+            openwisp_controller.config.tests.test_config \
+            openwisp_controller.config.tests.test_controller \
+            openwisp_controller.config.tests.test_device \
+            openwisp_controller.config.tests.test_template \
+            openwisp_controller.config.tests.test_views \
+
```

Comment:
These tests will always gets skipped while running tests using molecule. Unless we make changes to the test suite and add separate cases where `openwisp2_monitoring` variable is set to false, this is not useful. 

I don't think we need to alter the test suite just to make these tests run since we will be fixing this in near future. What do you think @Purhan ?

Reply:
I was thinking that another possible solution is to change those specific tests because the reason they fail is stupid (eg they expect 1 template but there are 2 because openwisp monitoring creates a template, this should be easy to fix in openwisp-controller.

Reply:
Created an issue for it: https://github.com/openwisp/openwisp-controller/issues/442

Reply:
So should I remove all these tests for the time being, so that the build passes?

Reply:
I looked into this now. Modifying the tests will take time. It may be more practical to just write some integration tests. So yes, please remove these.